src/mono/wasm/runtime/library_mono.js

```diff
@@ -1700,6 +1709,8 @@ var MonoSupportLib = {
 		//      "invariant": operate in invariant globalization mode.
 		//      "auto" (default): if "icu" behavior assets are present, use ICU, otherwise invariant.
 		//    diagnostic_tracing: (optional) enables diagnostic log messages during startup
+		//    custom_marshalers: (optional) a dictionary-style Object. keys identify managed types by 
+		//      FullName, and values specify the FullName of a marshaler type responsible for them.
```

Comment:
FullName or Assembly Qualified Name?

Reply:
It's FullName right now, do you think it should be assembly qualified? I'm not sure that adds any value

Reply:
I think both key and value should be assembly qualified.  Could imagine users creating source generators for their code that spit out a JS marshaller for their own types potentially having clashing names.

Reply:
Sounds good, I'll figure out the necessary changes for that

Reply:
OK, assembly-qualified names are now supported but not required, and I updated the builtins to assembly-qualify their names